go-runtime/compile/parser.go

```diff
@@ -199,23 +242,27 @@ func parseDirectives(node ast.Node, fset *token.FileSet, docs *ast.CommentGroup)
 			continue
 		}
 		pos := fset.Position(line.Pos())
+		ppos := schema.Position{
+			Filename: pos.Filename,
+			Line:     pos.Line,
+			Column:   pos.Column + 2, // Skip "//"
+		}
+
 		// TODO: We need to adjust position information embedded in the schema.
```

Comment:
I think this old TODO can be removed now

Reply:
Ahh of course! :)

Reply:
`Adjust the Participle-reported` comment too!